config/config.go

```diff
@@ -19,7 +18,7 @@ type Optimus struct {
 	Project    Project      `mapstructure:"project"`
 	Namespaces []*Namespace `mapstructure:"namespaces"`
 
-	Server    ServerConfig    `mapstructure:"serve"`
+	Server    Serve           `mapstructure:"serve"`
```

Comment:
Will we keep this struct or it will be deleted later ?

Reply:
`Serve` struct will not be deleted, only `Optimus` struct that will be deleted